### DISCUSSION TITLE
No visible indication of missing newline at end of file

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#139953] [Mobile]
GitHub Mobile beta for Android 

Open https://github.com/argoproj/argo-cd/commit/0fe0602a899d9fd439535c4b6c38ef5cc475f89b

See ![Screenshot_20240925-131155.png](https://github.com/user-attachments/assets/95f609f1-bcac-42b3-8625-674de824ba3a)

Browser is just as unhelpful:
![Screenshot_20240925-131343.png](https://github.com/user-attachments/assets/a7f9071f-5940-4690-87a2-2fcaabc055e3)

![Screenshot_20240925-131417.png](https://github.com/user-attachments/assets/9710bc5a-937f-410c-a761-6564f8f20326)

There's clearly a change somewhere...

This appears to be a bug in both Mobile and browser, but...

**Comment by HenSquared** [comment]
@jsoref,
Can you provide a bit more details on the concern? When I open Action.lua, I see there is a new line at the end of the file. 

![image](https://github.com/user-attachments/assets/3df41a79-f206-4dff-a8a4-abf842fc94fe)

**Reply by jsoref** [reply]
Normally when there's a diff, it's a change between two things, and it _shows_ that change.

Here's what I see when I ask a cli to explain https://github.com/argoproj/argo-cd/commit/0fe0602a899d9fd439535c4b6c38ef5cc475f89b:
```diff
git show 0fe0602a899d9fd439535c4b6c38ef5cc475f89b -- resource_customizations/apps/{DaemonSet,Deployment,StatefulSet}/actions/scale/action.lua
commit 0fe0602a899d9fd439535c4b6c38ef5cc475f89b
Author: ratulbasak <ratulbasak93@gmail.com>
Date:   Wed Sep 25 17:55:18 2024 +0200

    Ratul Basak ratulbasak93@gmail.com Updated Lua scripts
    
    Signed-off-by: ratulbasak <ratulbasak93@gmail.com>

diff --git a/resource_customizations/apps/DaemonSet/actions/scale/action.lua b/resource_customizations/apps/DaemonSet/actions/scale/action.lua
index cf593396f..172d17640 100644
--- a/resource_customizations/apps/DaemonSet/actions/scale/action.lua
+++ b/resource_customizations/apps/DaemonSet/actions/scale/action.lua
@@ -1,4 +1,4 @@
 local os = require("os")
 
 obj.spec.replicas = tonumber(scale)
-return obj
\ No newline at end of file
+return obj
diff --git a/resource_customizations/apps/Deployment/actions/scale/action.lua b/resource_customizations/apps/Deployment/actions/scale/action.lua
index cf593396f..172d17640 100644
--- a/resource_customizations/apps/Deployment/actions/scale/action.lua
+++ b/resource_customizations/apps/Deployment/actions/scale/action.lua
@@ -1,4 +1,4 @@
 local os = require("os")
 
 obj.spec.replicas = tonumber(scale)
-return obj
\ No newline at end of file
+return obj
diff --git a/resource_customizations/apps/StatefulSet/actions/scale/action.lua b/resource_customizations/apps/StatefulSet/actions/scale/action.lua
index cf593396f..172d17640 100644
--- a/resource_customizations/apps/StatefulSet/actions/scale/action.lua
+++ b/resource_customizations/apps/StatefulSet/actions/scale/action.lua
@@ -1,4 +1,4 @@
 local os = require("os")
 
 obj.spec.replicas = tonumber(scale)
-return obj
\ No newline at end of file
+return obj
```

Note the:
> \ No newline at end of file

Which is missing from the pictured content.
